### ISSUE TITLE
Suggestion: add a disclaimer to resolve confusing terminology inherited from Thunderbird

### ISSUE THREAD

**Issue by bertvandepoel** [cleidigh/Localfolder-TB#44]
Due to the confusing terminology used within Thunderbird (see https://bugzilla.mozilla.org/show_bug.cgi?id=1699984 which I just filed as well), I was under the impression I could use your addon to add a local mbox or maildir created by Postfix. It seems that's not the case, since the local folders API in thunderbird is only meant for Thunderbirds own file format and expectations.

Would you maybe consider adding a disclaimer on the addon page and in your README here that explains that your addon sadly cannot import independent mailbox in mbox/maildir format that don't originate from Thunderbird? That would probably prevent other users the confusion and waste of time if they're trying to do the same thing I was trying. Hopefully Thunderbird will also clarify this based on my report.

**Comment by LorenAmelang** [comment]
bertvandepoel, Have you tried copying the content of the Postfix folder you want to "adopt" into a TBird Local Folder, and then:

Locate the folder which you wish to repair.
Right click on it & select properties.
Go to General information option.
Now click Repair folder & click ok.

This recreates the .msf file for the folder, or creates one for folders that were not created by Thunderbird. 
It seems to catch almost all EverDesk Maildir files. 
But it found no information beyond Size for the "Fwd" Co-Op receipt messages that were invisible as originally received, and forwarded to myself in EverDesk. All the times show the hour and minute of the "Repair". No way to identify message Subjects, just the cryptic TBird hashes, but the content is all there, and they are searchable!

**Comment by bertvandepoel** [comment]
Thank you for the suggestion. In my case, I have a local Postfix for development purposes. So I was hoping to keep track of new emails for the connected local mailbox in Thunderbird. I don't think it will correctly process new emails after I use the repair functionality, or what do you think? I'm probably going to fold and run a local IMAP server using dovecot as well, it just seems a bit silly.

**Comment by LorenAmelang** [comment]
Suspect you're right. My use was a one-time import, it definitely doesn't update itself if the old source folder changes.

**Comment by bertvandepoel** [comment]
Yeah, it's sad but I understand why. The terminology Thunderbird uses just made it seem as if it was possible, so that's why I'm trying to have it changed.

**Comment by sciancio** [comment]
Hi LorenAmelang,have you ever thought to expand this addon to read from a unix mbox file? The "Unix Movemail" feature was removed from the last Thunderbird and this addon could be substitute it.

**Comment by LorenAmelang** [comment]
@sciancio I'm not the developer, just another user. But if you only need a one-time import from unix, I suspect the strategy I used would work.
